src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -99,11 +99,12 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
             throw new RuntimeException('Either a "route_name" or a "method" operation attribute must exist.');
         }
 
+        $actionName = sprintf('%s_%s', strtolower($operation['method']), $collection ? 'collection' : 'item');
+        $controller = self::DEFAULT_ACTION_PATTERN.$actionName;
+
         if (isset($operation['controller'])) {
             $controller = $operation['controller'];
-        } else {
-            $actionName = sprintf('%s_%s', strtolower($operation['method']), $collection ? 'collection' : 'item');
-            $controller = self::DEFAULT_ACTION_PATTERN.$actionName;
+            $actionName .= $operation['controller'];
```

Comment:
Not sure about this resulting in the best name, before this was defined through the `route_name` parameter.

@see: https://github.com/api-platform/core/blob/1.x/Api/Operation/OperationFactory.php#L156